atomic_reactor/plugins/post_flatpak_create_oci.py

```diff
@@ -81,6 +83,15 @@ def build_flatpak_image(self, source, build_dir: BuildDir) -> Dict[str, Any]:
         metadata = get_exported_image_metadata(outfile, IMAGE_TYPE_OCI)
         metadata['ref_name'] = ref_name
 
+        cmd = ['skopeo', 'copy', 'oci:{path}:{ref_name}'.format(**metadata), '--format=v2s2',
+               'docker-archive:{}'.format(str(build_dir.exported_squashed_image))]
```

Comment:
Oh, it gets recreated in the next commit :D

Do we want to remove the `outfile` after this, or is that used for something later?

Reply:
Also, what about the returned metadata? That's still for the oci image, not the v2s2 one?

Reply:
yes, it's still OCI. I didn't change it to `v2s2` metadata since we are only doing `v2s2` to be able to push the image in a format that the registry supports and it's also not possible to get layer sizes from OCI.

Reply:
I think it's possible, but we would have to update the imageutil method to handle OCI. But there might be more changes needed, so saving it as v2s2 is probably better.

What about removing the outfile? Worth it or no?

Reply:
we can remove outfile & outfile_tarred I'd say 

since we have already metadata and image in v2s2 ....

Reply:
unfortunately we cannot remove the outfile since it's the directory that contains the OCI image that is actually pushed by `tag_and_push`.

Reply:
ic, well we could remove outfile, and just push this v2s2 in tag_and_push plugin

but when everything supports oci images and we would like to push really oci image to registry we would have to revert that change again

so just leave outfile there

Reply:
Hmm, so wait a minute, what is oci and what is v2s2 in the flatpak workflow? We push the oci image and use metadata for the oci image. The v2s2 image is used for getting layer sizes and what else?

This commit also sets the image type to v2s2 for some image (not quite sure what image that is)
https://github.com/containerbuildsystem/atomic-reactor/blob/e10bc1de6635764a4d8663239721b467e52b2bf6/atomic_reactor/utils/koji.py#L517-L518

Reply:
I see. Looking closer, it probably doesn't matter what we pass here because the "image" from get_image_output is not an image at all, just the dummy tarball with the README, so v2s2 or oci doesn't really matter